packages/suite/src/views/dashboard/PromoBanner.tsx

```diff
@@ -241,27 +236,31 @@ export const PromoBanner = () => {
             )}
 
             <MobilePromoContainer>
-                <Translation
-                    values={{ b: text => <b>{text}</b> }}
-                    id="TR_MOBILE_APP_PROMO_TEXT_FOOTER"
-                />
-
-                <BadgeContainer>
-                    <StoreBadge
-                        url={SUITE_MOBILE_APP_STORE}
-                        image="APP_STORE"
-                        type="app-store"
-                        analyticsPayload="ios"
-                        shownQRState={shownQRState}
-                    />
-                    <StoreBadge
-                        url={SUITE_MOBILE_PLAY_STORE}
-                        image="PLAY_STORE"
-                        type="play-store"
-                        analyticsPayload="android"
-                        shownQRState={shownQRState}
+                <Row justifyContent="space-between" width="100%">
+                    <Translation
+                        values={{ b: text => <b>{text}</b> }}
+                        id="TR_MOBILE_APP_PROMO_TEXT_FOOTER"
                     />
-                </BadgeContainer>
+
+                    <Row>
+                        <BadgeContainer>
```

Comment:
Does it work without using `BadgeContainer`?

(Row and BadgeContainer are defacto duplicity here)

Reply:
I will try